cmake/DetectCodeCoverage.cmake

```diff
@@ -20,7 +26,8 @@ if ( GCOV AND
      LCOV AND
```

Comment:
With https://github.com/quinoacomputing/quinoa/pull/108/commits/ac5d540031ef3c519c756af5ca1772158141bcfe we no longer inject `-O0`.